MultiSource/Benchmarks/Ptrdist/anagram/anagram.c

```diff
@@ -262,42 +273,71 @@ void Fatal(char *pchMsg, unsigned u) {
  * byte streams are concatenated, and terminated with a 0.
  */
 
-void ReadDict(char *pchFile) {
-    FILE *fp;
-    char * pch;
-    char * pchBase;
+void ReadDict(_Ptr<char> pchFile) {
+    _Ptr<FILE> fp = 0;
+    // CHECKEDC : automatic checkedc pointer value initializer
```

Comment:
Thanks for recommendation
I have noticed that the it can be problem to use global checked pointer variable to represent bounds of local variable since it can be modified in function call
I will apply your recommendation for next pull request